Update: Move Partner Secret Form Field to Initial Screen

jennsun opened the pull request:
https://app.clickup.com/t/xfj00b

Moved the "partner secret" form field from the second page's “waymark.createAccount()” and “waymark.loginAccount()” forms to the first page.

jennsun committed:
Update: Move Partner Secret Form Field to Initial Screen

simanthastew commented on example_site/src/components/AccountAuthentication.js:
```diff
@@ -37,9 +37,13 @@ function LoginAccountForm() {
-  const onSubmit = async ({ privateKey, accountID, externalID }) => {
+  const onSubmit = async ({ accountID, externalID }) => {
+    
+    console.log("LOGIN PRIVATE KEY:", partnerSecret);
```
Looks like you added a few log statements, let's remove those before merging! They are a super useful tool for debugging (and I know we have a few sprinkled in since it's a demo site), but typically you wouldn't want to log out something like a parter secret in the console because anyone could see it.

simanthastew commented on example_site/src/components/AccountAuthentication.js:
```diff
@@ -129,17 +120,21 @@ function CreateAccountForm() {
+      console.log("Create Account FORM DATA", formData);
+      console.log("Create Account Secret", partnerSecret);
+
       const { privateKey } = formData;
```
Is the `privateKey` variable being used anywhere? And does `formData` still have a `privateKey` field?

simanthastew commented on example_site/src/components/AccountInfoForm.js:
```diff
@@ -6,7 +6,6 @@ export default function AccountInfoForm({
   account,
   formTitle,
   onSubmit,
-  shouldRequirePrivateKey,
```
Are any parent components passing this prop value into this component?

simanthastew reviewed (changes requested):
Really nice work! Just a few small things to clean up

jennsun committed:
Update: PR Review
removed log statements, removed parent components that passed shouldRequirePrivateKey prop
